source/zh-cn/docs/commands.md

```diff
@@ -155,11 +179,26 @@ $ hexo --silent
 
 ### 自定义配置文件的路径
 
-``` bash
-$ hexo --config custom.yml
+```bash
+# use 'custom.yml' in place of '_config.yml'
```

Comment:
comments should be translated as well.

Reply:
The translations of those comments were added in https://github.com/hexojs/site/pull/1030/commits/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe